newton/_src/viewer/recording.py

```
@@ -0,0 +1,376 @@
+                    pass
+                try:
+                    self._queue.put_nowait(None)
+                except queue.Full:
+                    pass
+        if self._worker is not None:
+            self._worker.join(timeout=5.0)
+            self._worker = None
+
+        try:
+            self._proc.wait(timeout=10.0)
+        except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)